pom.xml

```diff
@@ -243,7 +243,6 @@
                         -Xms128m -Xmx1G
                         -Dhazelcast.phone.home.enabled=false
                         -Dhazelcast.mancenter.enabled=false
-                        -Dhazelcast.logging.type=none
```

Comment:
Why is this change needed?

Reply:
To see the log output in tests and make bug fixing easier